src/MAUI/Maui.Samples/Samples/Layers/RasterLayerRasterFunction/RasterLayerRasterFunction.xaml.cs

```diff
@@ -112,7 +112,7 @@ private async Task Initialize()
                 // Zoom the map to the extent of the image service raster (which also the extent of the raster layer)
                 await MyMapView.SetViewpointGeometryAsync(myArcGISImageServiceInfo.FullExtent);
 
-                // NOTE: The sample zooms to the extent of the ImageServiceRaster. Currently the ArcGIS Runtime does not
+                // NOTE: The sample zooms to the extent of the ImageServiceRaster. Currently the ArcGIS Maps SDK for .NET does not
```

Comment:
If this general statement across all SDKs then we should say ArcGIS Maps SDK for Native Apps